p0074_slon_cpp

```diff
@@ -1,7 +1,8 @@
-@include <iostream>
+#include <iostream> // 1/2
```

Comment:
- [ ] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`